tests/test-cases/Group24-Host-Affinity/24-01-Basic.robot

```diff
@@ -0,0 +1,87 @@
+# Copyright 2018 VMware, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#	http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License
+
+*** Settings ***
+Documentation     Suite 24-01 - Basic
+Resource          ../../resources/Util.robot
+Test Teardown     Cleanup
+Default Tags
+
+
+*** Keywords ***
+Cleanup
+    Remove Group     %{VCH-NAME}
+
+    Cleanup VIC Appliance On Test Server
+
+Remove Group
```

Comment:
Is it a good idea to have a Group24 util and move group cleanup and related keywords there? Or we can do that after we add more complicated tests

Reply:
We can refactor common code into an appropriate place (e.g., a Group24 util) when we re-use it.